### ISSUE TITLE
INSUFFICIENT_OUTPUT_AMOUNT error

### ISSUE THREAD

**Issue by yuichiroaoki** [yuichiroaoki/poly-flash#4]
ref.
https://docs.uniswap.org/protocol/concepts/V3-overview/swaps#safety-checks
https://ethereum.stackexchange.com/questions/90241/why-insufficient-output-amount

**Comment by hatemua** [comment]
I found a solution to this problem 
when you increase the slipage to 5 and the gas price 90 GWEI , it write to polyscan

**Comment by rolandweb3** [comment]
> I found a solution to this problem when you increase the slipage to 5 and the gas price 90 GWEI , it write to polyscan

How to start the bot?

**Comment by hatemua** [comment]
I test it and I think it has over 3 years without any changes , and with bugs not solved

**Comment by rolandweb3** [comment]
> I test it and I think it has over 3 years without any changes , and with bugs not solved

Yes I successfully deployed everything but when I run the flashloan.ts it's always giving me the same txn hash, the txn is not even in the block explore

**Comment by rolandweb3** [comment]
Have you found a solution
